style-desk.css

```diff
@@ -0,0 +1,603 @@
+  }
+
+  .headline .head h1 {
+    color: #ff6b00;
+    font-family: 'Crete Round', serif;
+    font-size: 56px;
+    line-height: 64px;
+    font-weight: 400;
+    letter-spacing: 0.37px;
+    margin: 0;
+  }
+
+  .headline .head h2 {
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+    color: #fff5e1;
+    font-weight: 700;
+    font-size: 24px;
+    line-height: 32px;
+  }
+
+  .headline .head p {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 24px;
+    color: #fff;
+  }
+
+  .headline .social {
+    order: 1;
+    list-style: none;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    height: 178px;
+  }
+
+  .works {
+    grid-row: 3/4;
+    margin-bottom: 100px;
+  }
+
+  .intro {
+    margin: 118px 9vw 50px 9vw;
+    display: grid;
+    width: auto;
+    grid-template-columns: 47.5vw 31vw;
+    grid-template-rows: 52px 420px;
+    row-gap: 120px;
+    column-gap: 24px;
+  }
+
+  .introhead {
+    grid-row: 1/2;
+    grid-column: 1/2;
+    font-family: 'Crete Round', serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 40px;
+    line-height: 52px;
+    color: #091e42;
+    margin-top: 0;
+  }
+
+  .intro .line {
+    grid-row: 1/2;
+    grid-column: 1/3;
+    width: 50vw;
+    height: 1px;
+    background-color: black;
+    margin: 30px 0 auto auto;
+  }
+
+  .intro img {
+    grid-row: 2/3;
+    grid-column: 1/2;
+    width: 47.5vw;
+    height: 448px;
+  }
+
+  .intro .introinfo {
+    grid-row: 2/3;
+    grid-column: 2/3;
+    width: 31vw;
+    height: 56vh;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+  }
+
+  .intro div h3 {
+    font-family: 'Crete Round', serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 40px;
+    line-height: 52px;
+    color: #091e42;
+    margin-top: 0;
+    margin-bottom: 0;
+  }
+
+  .intro div .introtext {
+    font-family: 'Inter', sans-serif;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 24px;
+    color: #091e42;
+  }
+
+  .intro div ul {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    width: 20vw;
+    gap: 12px;
+    padding-left: 0;
+  }
+
+  .intro div ul li {
+    margin-left: 0;
+    list-style: none;
+    text-align: center;
+    font-family: 'Inter', sans-serif;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #091e42;
+    border: 1px solid #8993a4;
+    padding: 10px 12px;
+  }
+
+  .intro .introbutton {
+    background-color: #ff6b00;
+    height: 48px;
+    width: 124px;
+    font-family: 'Inter', sans-serif;
+    font-weight: 600;
+    font-size: 17px;
+    line-height: 24px;
+    text-align: center;
+    align-items: center;
+    letter-spacing: 0.03em;
+    color: #fff;
+    margin-bottom: 24px;
+    border: none;
+  }
```

Comment:
Please make sure you add the interaction stylings to all of your buttons. You should fix the button height as well.